CHANGELOG.md

```diff
@@ -5,28 +5,37 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+* New HDF5 support
 
 ## [0.1.3]
 ### Added
 * Pass save results as parameter for node
 
 ## [0.1.2] - 2021-06-08
-
 ### Added
 * Pass save results as parameter for node
```

Comment:
duplicate for 0.1.3
was it added there or here?

Reply:
there is no 0.1.3 :|